docker/dev/docker-compose.yml

```diff
@@ -20,14 +26,14 @@ services:
     env_file:
       - docker-compose.env
     restart: always
-  mysql_up:
-    image: alpine
-    container_name: mysql_up
-    command: sh -c "while ! nc -z mysql 3306 > /dev/null; do echo waiting for mysql; sleep 3; done"
-    env_file:
-      - docker-compose.env
+    healthcheck:
```

Comment:
NICE!

Reply:
yeah, I updated compose to use healthchecks, since they're available, i don't see a reason for us to not be using them. or creating a container that does a health check. also the previous implementation would run forever. and we definitely want something that would timeout.